python/flashinfer/gemm.py

```diff
@@ -194,3 +196,32 @@ def run(
         )
 
     forward = run
+
+
+def bmm_fp8(
+    A: torch.Tensor,
+    B: torch.Tensor,
+    D: torch.Tensor,
+    A_scale: torch.Tensor,
+    B_scale: torch.Tensor,
+):
+    r"""BMM FP8
+
+    Parameters
+    ----------
+    A: torch.Tensor
+        Input tensor, shape (b, m, k).
+
+    B: torch.Tensor
+        Mat2 tensor, shape (b, k, n), should be column major.
+
+    D: torch.Tensor
```

Comment:
I prefer to make it an optional argument 'out', if not specified, we created an empty tensor with output shape, otherwise we update out's value in-place, and this function should return 'out'.

We don't have to modify code on cpp side, just do them in python.

Also, please mention the input (e4m3) and output (bf16) data type explicitly in docstring.

Reply:
ok